cocos2d/render-texture/CCRenderTextureCanvasRenderCmd.js

```diff
@@ -88,6 +88,7 @@
 
         var scale = cc.contentScaleFactor();
         cc.renderer._renderingToCacheCanvas(this._cacheContext, node.__instanceId, scale, scale);
+        node.sprite._renderCmd.setDirtyFlag(cc.Node._dirtyFlags.colorDirty);
```

Comment:
In that case, we should overwrite the `updateStatus` & `_syncStatus` of RenderTexture's canvas & webgl render command, update the textureDirty flag in them.